catalog/opensearch/catalog-opensearch-source/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -71,6 +71,10 @@
             name="Receive Timeout" id="receiveTimeout"
             required="true" type="Integer" default="60000"/>
 
+        <AD description="Poll Interval to Check if the Source is available (in minutes - minimum 1)."
```

Comment:
✏️ capitalization of words in mid sentence

Reply:
Interesting that we use minutes for this interval and milliseconds for other intervals.  I wonder if there is a case to use milliseconds to provide administrators with greater granularity.

Reply:
It looks like we use minutes for the other sources though

Reply:
Should we be consistent everywhere?

Reply:
@bdeining millisecond polling would be odd

Reply:
I'm not suggesting we poll on milliseconds, but give the ability to poll every 6.5 minutes

Reply:
I don't think it's worth it without a demonstrated need

Reply:
This is exactly what makes seamless upgrade hard when later we have to change the type or the meaning of configuration parameter.

Reply:
@paouelle Can you elaborate?

Reply:
When trying to support seamless upgrades, the more configuration properties that have changed meaning between releases (e.g. changed from minutes to seconds) means we have to write additional code to special handle those. My take would be to make everything in seconds which is granular enough to easily account for all cases we can think of now and in the future. I highly doubt one would want to poll a source every milliseconds (but again I could be wrong in which case we should define it in milliseconds right now). If we think it makes sense to support half minutes than my vote is to go to seconds which is more user friendly than 1.5 or 1.3 minutes.

It is also one of the reason I have never been a fan of driving the UI from the storage format.